server/db/models/Product.js

```diff
@@ -0,0 +1,71 @@
+const db = require("../db");
+const { STRING, INTEGER, DECIMAL, TEXT } = require("sequelize");
+
+///////////////// PRODUCT MODEL /////////////////
+const Product = db.define("product", {
+  name: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  // Note: the brand column will be used to pair Product with Brand via script/seed.js.
+  // john - I comment this out because this is automatically create by the association in the indexjs
+  brandName: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  imageURL: {
+    type: STRING,
+    allowNull: false,
+  },
+  price: {
+    type: INTEGER,
+    allowNull: false,
+  },
+  category: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  gender: {
+    type: STRING,
```

Comment:
if you believe in "2 genders" you can do like BOOL
if you dont, you can do `ENUM(['MALE', 'FEMALE'])`